### PR TITLE
Add The Red Programming Language

### PR DESCRIPTION
**Pull request by hako**
Red Programming Language is a promising high level programming language inspired by Rebol with built-in GUI support. It is in active development and deserves recognition.

Note: I put this in a separate PR and closed the older one.

### CODE REVIEW

File: engineering_blogs.opml [hunk 1]
```diff
@@ -115,7 +115,6 @@
       <outline type="rss" text="Flickr" title="Flickr" xmlUrl="http://code.flickr.net/feed/" htmlUrl="http://code.flickr.net/"/>
       <outline type="rss" text="Flipboard" title="Flipboard" xmlUrl="http://engineering.flipboard.com/feed.xml" htmlUrl="http://engineering.flipboard.com/"/>
       <outline type="rss" text="Foursquare" title="Foursquare" xmlUrl="https://engineering.foursquare.com/feed" htmlUrl="https://engineering.foursquare.com/"/>
-      <outline type="rss" text="Freeletics" title="Freeletics" xmlUrl="https://freeletics.engineering/feed.xml" htmlUrl="https://freeletics.engineering/"/>
```
Comment (wcerfgba) [inline engineering_blogs.opml]: This should not have been removed

File: README.md [hunk 2]
```diff
@@ -764,6 +764,7 @@
 #### R technologies
 * React https://facebook.github.io/react/blog/
 * React Native http://facebook.github.io/react-native/blog/
+* Red http://www.red-lang.org
```
Comment (wcerfgba) [inline README.md]: Can we change this to 'Red Programming Language', to disambiguate the entry?

File: engineering_blogs.opml (hunk 1 above)
Comment (hako) [reply engineering_blogs.opml]: Freeletics's Engineering blog isn't on the README, thats probably why it was removed. A separate PR to include Freeletics again should resolve this, what do you think?

Comment (wcerfgba) [reply engineering_blogs.opml]: Sounds good, would you like to submit it? 😄

Comment (hako) [reply engineering_blogs.opml]: Sure! 👍  I'll make a separate PR Freeletics soon.

**Pull request merged**